tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES2022",
+    "target": "ES2023",
```

Comment:
won't this make the output not compatible with anything < es2023

Reply:
According to [MS](https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping): `All versions of Node 18 support all ES2023 runtime features, so lib can be safely set to ES2023.`

Reply:
I do not have a big need for `ES2023` so we can keep it at `ES2022` if you like.